inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
Can we avoid this? enable makes the most sense, this will become a mess over time.